clang/test/CodeGenSYCL/intel-sycl-ptr-annotation-builtin.cpp

```diff
@@ -0,0 +1,140 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-linux -std=c++11 -disable-llvm-passes -S -opaque-pointers -emit-llvm -x c++ %s -o - | FileCheck %s
```

Comment:
Please apply to all run lines in all clang tests
```suggestion
// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-linux -emit-llvm %s -o - | FileCheck %s
```

Reply:
I don't think this is resolved?

Reply:
Tested. The -disable-llvm-passes  need to preserve. other can be removed